### ISSUE TITLE
insert the function proj_from_CF_dict() in Unstructured netCDF4 reader or basereader

### ISSUE THREAD

**Issue by Boorhin** [OpenDrift/opendrift#785]
Hi,
I just noticed that this handy function that could read the attribute of projection of the netcdf exists for the structured reader and not the unstructured reader. I believe it could come handy in both?

Maybe it could be included in the base reader so that there is no repetition?

Cheers,

**Comment by knutfrode** [comment]
Yes, you are right. Instead of duplicating, this should be moved up to basereader. I will do this soon, you may ping this issue if nothing happens.
